### ISSUE TITLE
Github contributions

### ISSUE THREAD

**Issue by majakudlicka** [FACG2/getGitHub#33]
Some members of your team have much more contributions than the others. You should try to spread the work more evenly and make sure you swap pairs and computers. The quality of your work as a team on the project is more important than the end result.

**Comment by Walidmash** [comment]
yes , Kefah laptop has a problem in pushing files , we spend more than hour trying to fix it, but we couldn't , so kefah sent us her codes through gitter, 
and yes we will put this note in mind in next project,
thank you Maja :)
